app/models/like.rb

```diff
@@ -1,2 +1,13 @@
 class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create :update_likes_counter
+  after_destroy :update_likes_counter
```

Comment:
Kindly I would recommend you here to also use after_save instead of after_create, because after_save is triggered on create and update while after_Create is only called on create.
Note: The same for comment.rb file